eng/Configurations.props

```diff
@@ -38,6 +37,10 @@
   </PropertyGroup>
 
   <PropertyGroup>
+    <HostRuntimeIdentifier Condition="'$(HostRuntimeIdentifier)' == '' and '$(MSBuildRuntimeType)' == 'core'">$([System.Runtime.InteropServices.RuntimeInformation]::RuntimeIdentifier)</HostRuntimeIdentifier>
+    <HostRuntimeIdentifier Condition="'$(HostRuntimeIdentifier)' == '' and '$(MSBuildRuntimeType)' != 'core'">win-$([System.Runtime.InteropServices.RuntimeInformation]::OSArchitecture.ToString().ToLowerInvariant)</HostRuntimeIdentifier>
+    <RuntimeOS Condition="'$(RuntimeOS)' == '' and '$(HostRuntimeIdentifier)' != ''">$(HostRuntimeIdentifier.Remove($(HostRuntimeIdentifier.LastIndexOf('-'))))</RuntimeOS>
```

Comment:
This should not be called `RuntimeOS`. The other places that use `RuntimeOS` in this repo use it as "The machine that we are targeting.". The meaning here is "The machine that are running on.".

Reply:
Also, I am not sure whether it makes sense to move these `Host*` properties to the shared file. They should not be ever used. Their use in the installer partition is less than ideal.

Reply:
> This should not be called RuntimeOS.

This is already called RuntimeOS in libraries. That property is currently generated in https://github.com/dotnet/arcade/blob/master/src/Microsoft.DotNet.Build.Tasks.TargetFramework.Sdk/src/AddRuntimeOSPropertyTask.cs#L12 and then imported in https://github.com/dotnet/runtime/blob/master/src/libraries/Directory.Build.props#L29.

For the sake of this PR I will move the `RuntimeOS` property into libraries.

> Also, I am not sure whether it makes sense to move these Host* properties to the shared file.

The `HostRuntimeIdentifier` property is used in both libraries and installer at the moment that's why I'm extracting the property into this shared file.

Reply:
> The other places that use RuntimeOS in this repo use it as "The machine that we are targeting.". The meaning here is "The machine that are running on.".

There is a property directly below this named `$(TargetOS)`, why don't we use that property to mean "The machine that we are targeting"? That would be a much better name for that property IMO.

Reply:
Note: `src\libraries` uses `RuntimeOS` to mean "The machine that we building on." That is the MSBuild Target that is being removed in this PR - see the `$(RuntimePropsFile)`.

Maybe it would make sense to rectify "RuntimeOS" in a follow up PR? That way we can make forward progress in getting the new SDK in.

Reply:
I moved the property into libraries where it was imported beefore to maintain the existing behavior and semantic. Let's follow-up on the nomenclature in an issue.